src/main.js

```diff
@@ -57,6 +58,36 @@ module.exports = {
     }))
 
     this.subscriptions.add(atom.commands.add('atom-text-editor', {
+      'linter-eslint:debug': () => {
+        const textEditor = atom.workspace.getActiveTextEditor()
+        const filePath = textEditor.getPath()
+        const linterEslintMeta = require(Path.join(atom.packages.resolvePackagePath('linter-eslint'), 'package.json'))
+        const config = atom.config.get('linter-eslint')
+        const configString = JSON.stringify(config, null, 2)
+        const hoursSinceRestart = process.uptime() / 3600
```

Comment:
I think tenths of hours (6 min resolution, which is what it is showing now) is enough to tell us whether it's been restarted recently, without being ridiculous when it hasn't been restarted in days.  But, happy to change it if you think minutes would be better.